<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChonyaAdd</class>
 <widget class="QMainWindow" name="ChonyaAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品出数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>材料规格:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具编号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>125</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品料号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MoldID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>315</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>输入编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>45</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>产品名称:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProductStand">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>195</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入材料规格</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>管制标准:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>245</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(170, 170, 255);
	border-radius: 15px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(170, 85, 255);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>增加模具</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutNum_1">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>315</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>315</y>
      <width>56</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutNum_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>315</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LoadingLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/logo110x30.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>666</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProductID">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>115</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品料号</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProductName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>请输入产品名称</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ControlStand">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>255</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <item>
     <property name="text">
      <string>NULL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格＜0.3mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格≧0.3且﹤0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>铜材材料厚度规格≧0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不锈钢材料厚度规格＜0.5mm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>不锈钢材料厚度规格≧0.5m</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="LifeNum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>66</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>375</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具寿命冲次数:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MachineNum">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>370</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(160, 175, 200);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>1#</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>375</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增加机台:</string>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>Label6</zorder>
   <zorder>Label3</zorder>
   <zorder>Label5</zorder>
   <zorder>Label2</zorder>
   <zorder>MoldID</zorder>
   <zorder>Label1</zorder>
   <zorder>ProductStand</zorder>
   <zorder>Label4</zorder>
   <zorder>AddButton</zorder>
   <zorder>OutNum_1</zorder>
   <zorder>Label7</zorder>
   <zorder>OutNum_2</zorder>
   <zorder>LoadingLabel</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>ProductID</zorder>
   <zorder>ProductName</zorder>
   <zorder>ControlStand</zorder>
   <zorder>LifeNum</zorder>
   <zorder>Label8</zorder>
   <zorder>MachineNum</zorder>
   <zorder>Label9</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ProductName</tabstop>
  <tabstop>ProductID</tabstop>
  <tabstop>ProductStand</tabstop>
  <tabstop>ControlStand</tabstop>
  <tabstop>MoldID</tabstop>
  <tabstop>OutNum_1</tabstop>
  <tabstop>OutNum_2</tabstop>
  <tabstop>LifeNum</tabstop>
  <tabstop>MachineNum</tabstop>
  <tabstop>AddButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
